packages/terra-framework-docs/src/terra-dev-site/doc/compact-interactive-list/Examples.2/FlowDirection.4.doc.mdx

```diff
@@ -0,0 +1,27 @@
+import VerticalFlow from './VerticalFlow?dev-site-examples';
+import HorizontalFlow from './HorizontalFlow?dev-site-examples';
+
+# Flow Direction
+
+### Vertical Flow
+
+By default, the items in the [Compact Interactive List](/components/cerner-terra-framework-docs/compact-interactive-list/about) component are read from top to bottom before moving to a new visual column.
+
+<VerticalFlow />
+
+#### Usage
+
+* The [rowHeight](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) property is required for the list items to use vertical flow. If not set, the default value is used.
+* The items are distributed evenly between all columns but the last one. The last column accommodates the remainder after the number of items is divided by the number of columns.
+* Ensure that the number of rows is not divisible by the [numberOfColumns](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) value minus 1 without a remainder, which leads to an empty last column.
```

Comment:
This isn't necessarily something a consumer _shouldn't_ do. It's just a kind of weird side effect of how it works.
```suggestion
* Note: If the number of rows is divisible by the [numberOfColumns](/components/cerner-terra-framework-docs/compact-interactive-list/about#compact-interactive-list-props-table) value minus 1 without a remainder, the last column will be empty.
```